asyncapi.yaml

```diff
@@ -0,0 +1,94 @@
+asyncapi: '2.0.0' # Go parser does not support 2.1.0 at this moment.
+info:
+  title: AsyncAPI Event-Gateway demo API
+  version: 1.0.0-alpha
+  description: This API lets users interact with an instance of the [AsyncAPI Event-Gateway](https://github.com/asyncapi/event-gateway).
+defaultContentType: application/json
+servers:
+  asyncapi-event-gateway-demo:
+    url: 'event-gateway-demo.asyncapi.org:28002'
```

Comment:
This is not available yet. In case we decide another name, we can always rename it later.

Reply:
.org and not .com to make it clear it is not commercial solution?

Reply:
I think yes, .org is explicitly saying we are an org and not a commercial product. However, nowadays this is not important anymore. I'm happy to change to whatever we prefer.
Would you prefer `.com` here?